### ISSUE TITLE
Li12-system ABACUS compare with VASP in PW base

### ISSUE THREAD

**Issue by dyzheng** [deepmodeling/abacus-develop#529]
## Describe Current Status and Possible Solution

we should use same parameters to test ABACUS in PW base with VASP program.

## Additional Context
Input files in ABACUS are:
[ABACUS_VASP_example.zip](https://github.com/deepmodeling/abacus-develop/files/7659041/ABACUS_VASP_example.zip)
Output files in ABACUS and VASP are:
[result_6wrap_10unwrap.zip](https://github.com/deepmodeling/abacus-develop/files/7659043/result_6wrap_10unwrap.zip)

**Comment by dyzheng** [comment]
1. number of k-point：vasp 1008 vs ABACUS 2352； 
2. STRESS time：vasp 35s vs ABACUS 2809.89s；
3. force time：vasp 35s vs ABACUS 394.61s；
4. self-consistant times and average time： vasp 47.28s * 15 times  vs ABACUS 557s * 11 times
5. energy cutoff: vasp 1300eV vs ABACUS 200Ry ~ 2600eV

**Comment by iProzd** [comment]
By the way, about the experiments setting, I ran these experiments on two machines (for wraps and unwraps repectively) with same CPUs, but I found one of the machines (the unwraps) performes much faster than the other one (~3h vs ~9h). It might be the IO efficiency that differed and nevermind. There should be no efficiency difference in two kind of samples. We can ignore the [~9h] performance showed in results of wrap samples and simply assume that all these samples running on ABACUS end in about 3 hours.
